services/src/main/java/io/druid/cli/PullDependencies.java

```diff
@@ -65,7 +65,47 @@
 
   private static final Set<String> exclusions = Sets.newHashSet(
       "io.druid",
-      "com.metamx.druid"
+      "com.metamx.druid",
+
+      // These should be included by druid and not pulled down in extensions
+      // Note to future developers: This list is hand-crafted and will probably be out of date in the future
+      // A good way to know where to look for errant dependencies is to compare the lib/ directory in the distribution
+      // tarball with the jars included in the extension directories.
+      //
+      // This list is best-effort, and might still pull down more than desired.
```

Comment:
why are things not "desired" ? An explanation on the problem and what causes it would be more helpful and just an off-handed comment about something not being desired.